src/Repository/AvailableFile.hpp

```diff
@@ -51,6 +52,8 @@ struct AvailableFile {
 
   FileType type;
 
+  BrokenDate updated;
```

Comment:
updated what? What does that mean? Maybe call it `update_date`. Does it need a time of day?

Reply:
But yeah, I can rename it of course.

Reply:
If you give it bad names, at least add API documentation.